code/game/objects/items/devices/education_chips.dm

```diff
@@ -0,0 +1,95 @@
+//This is for electronic versions of learning, for now solely martial arts
+
+/obj/item/clothing/head/AED
+  name = "Automatic Education Device"
+  desc = "A complex assortment of metal, electrodes, and wires. Simply put the headset on, insert an education chip, and turn it on to instantly learn the contents of said chip."
+  icon_state = "electrode_helmet"
+  actions_types = list(/datum/action/item_action/instant_learn)
+  pocket_storage_component_path = /datum/component/storage/concrete/pockets/tiny/AED
+
+/obj/item/clothing/head/AED/item_action_slot_check(slot)
+  if(slot == SLOT_HEAD)
+    return 1
+
+/obj/item/clothing/head/AED/ui_action_click(mob/user, action)
+  educate()
+
+/obj/item/clothing/head/AED/verb/educate()
+  set src in usr
+  if(!src.contents.len)
+    to_chat(usr, "<span class='warning'>There is nothing loaded in the [src]!</span>")
+    return
+  for(var/obj/item/education_chip/E in src.contents)
+    if(!E.subject)
+      to_chat(usr, "<span class='warning'>The Education Chip doesn't have a subject!</span>")
+      return
+    var/datum/martial_art/MA = new E.subject
+    if(usr.mind.has_martialart(initial(MA.id)))
+      to_chat(usr,"<span class='warning'>You already know [E.subject_name]!</span>")
+      return
+    to_chat(usr, "<span class='notice'>Your head whirls with a sudden rush of information as you instantly learn about [E.subject_name]!</span>")
+    to_chat(usr, "[E.greet]")
+    MA.teach(usr)
+    usr.log_message("learned the subject [E.subject_name] ([MA])", LOG_ATTACK, color="orange")
+    qdel(E)
+    new /obj/item/education_chip(src)
+
+
+////Education Chips////
+
+/obj/item/education_chip
+  name = "Empty Education Chip"
+  desc = "An empty education chip."
```

Comment:
chances are traitors will trash or otherwise discard these when used, new players should probably know what these do beyond "hey look an empty chip??"

Reply:
Put a little more info there, but I don't want to add "suspicious-looking" or anything blatant. Possibly one day could be utilized for teaching skills or something idk.

Reply:
makes sense to me, i like it